test/quality-level-list.test.js

```diff
@@ -17,32 +16,34 @@ QUnit.test('Properly adds QualityLevels to the QualityLevelList', function(asser
     addCount++;
   });
 
-  this.qualityLevels.addQualityLevel(this.levels[0]);
+  let expected0 = this.qualityLevels.addQualityLevel(this.levels[0]);
 
   assert.equal(this.qualityLevels.length, 1, 'added quality level');
   assert.equal(addCount, 1, 'emmitted addqualitylevel event');
-  assert.strictEqual(this.qualityLevels[0], this.levels[0], 'can access quality level with index');
+  assert.strictEqual(this.qualityLevels[0], expected0, 'can access quality level with index');
 
-  this.qualityLevels.addQualityLevel(this.levels[1]);
+  let expected1 = this.qualityLevels.addQualityLevel(this.levels[1]);
 
   assert.equal(this.qualityLevels.length, 2, 'added quality level');
   assert.equal(addCount, 2, 'emmitted addqualitylevel event');
-  assert.strictEqual(this.qualityLevels[1], this.levels[1], 'can access quality level with index');
+  assert.strictEqual(this.qualityLevels[1], expected1, 'can access quality level with index');
 
-  this.qualityLevels.addQualityLevel(this.levels[0]);
+  let expected3 = this.qualityLevels.addQualityLevel(this.levels[0]);
```

Comment:
At first I thought the expected vars were references to their respective level indices, but when I saw this matched index 0, it threw me off. Maybe better to name something like expectedDupe?